index.html

```diff
@@ -389,11 +389,17 @@ <h2>
           <a>current settings object</a>.
         </li>
         <li>
-          Let <var>context</var> be the <var>document</var>'s <a>browsing context</a>.
-        </li>
-        <li>
-          If <var>context</var> is <code>null</code>, throw a
-          <code>"<a>NotSupportedError</a>"</code> <a>DOMException</a>.
+          If <a>global object</a> is not <a>DedicatedWorkerGlobalScope</a>,
+          run the following steps:
```

Comment:
These steps should still occur in a worker, but refer to the document owning the worker.